config/environment.rb

```diff
@@ -3,3 +3,13 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+ActionMailer::Base.smtp_settings = {
+  :user_name => 'apikey',
+  :password => ENV['SENDGRID_API_KEY'],
+  :domain => ENV["DOMAIN"] || "http://localhost:3000",
+  :address => 'smtp.sendgrid.net',
+  :port => 587,
+  :authentication => :plain,
+  :enable_starttls_auto => true
+}
```

Comment:
Could we move this into an initializer?

Reply:
The standard, according to the docs (https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration-for-gmail), is to place it in the environment file. I put it in `config/environment.rb` because I didn't see a need to differentiate between environments at this point, but it could be put in `config/environments/production.rb` to properly conform to the standard.

What's your reasoning for putting it into an initialiser?

Reply:
That's fine. I consider `environment.rb` to be for things that are specifically related to the environment (think Separation of Concerns). `environment.rb` isn't really concerned with setting up the mailer, and the initializers are specifically build to initialize things. So that's where I prefer to put this kind of code.

But it isn't really a big deal either way, and I don't want this to hold up merging.